### ISSUE TITLE
Cannot Delete WorkSpace

### ISSUE THREAD

**Issue by hossko** [coder/coder#6687]
Version: [`v0.19.2+7eb2c2f`](https://github.com/coder/coder/commit/7eb2c2ff6d229619f88f41e9b2b185714a967619)

Hello,

I cannot delete a workspace from the WebUI that have failed upon creation. From the CLI it works fine.

**Comment by sharkymark** [comment]
@hossko is it related to this issue #5986 ? You have to click a specific part of the delete UI for it to work.

**Comment by hossko** [comment]
Hello @sharkymark,

i tried clicking several parts of the Delete button, but not working at all. FYI, i can delete the workspace from the CLI.

**Comment by kylecarbs** [comment]
@hossko I'm unable to reproduce on our deployment :(

Could you take a screen recording?

**Comment by sharkymark** [comment]
@hossko Go into Chrome, Inspect (or similar browser tool) and are errors showing up when you click?

**Comment by hossko** [comment]
No, the inspect console does not show any kind of error. Are there logs on the server we can check ?

**Comment by kylecarbs** [comment]
Tracking in #5986
